scripts/preprocessing/04b-run_ssp.py

```diff
@@ -52,11 +78,13 @@ def run_ssp(subject, session=None):
 
     raw = mne.io.read_raw_fif(raw_fname_in)
     # XXX : n_xxx should be options in config
+    reject_eog, reject_ecg = _get_global_reject_ssp(raw)
+
     msg = 'Computing SSPs for ECG'
     logger.debug(gen_log_message(message=msg, step=4, subject=subject,
                                  session=session))
     ecg_projs, _ = compute_proj_ecg(raw, n_grad=1, n_mag=1, n_eeg=0,
```

Comment:
the number of projectors is hard coded to 1 here (and to 0 for EEG!). This makes sense in the context of where this code came from, namely, processing the Cam-CAN, but we should probably allow to set these via the config.